shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/presentation/StepQuizResolver.kt

```diff
@@ -60,4 +61,7 @@ object StepQuizResolver {
             else ->
                 false
         }
+
+    fun isQuizSupportable(blockName: String): Boolean =
+        BlockName.supportedBlocksNames.contains(blockName)
```

Comment:
It's better to pass a `Step` as a parameter of a function. When calling a function with a `block: String` parameter it's not easy to understand what to pass.